package.json

```diff
@@ -1,7 +1,14 @@
 {
   "name": "raster",
-  "version": "4",
+  "version": "6.0.0",
   "description": "Simple grid system",
+  "style": "raster.grid.css",
+  "main": "raster.grid.css",
+  "files": [
+    "raster.grid.css",
+    "raster.css",
+    "raster.debug.css"
+  ],
```

Comment:
Since version 20 some of the files have changed names.

- raster.grid.css — only the grid CSS
- raster2.css — full version of Raster (include the grid CSS; no need for raster.grid.css)
- raster2-react.d.ts — React type defs (includes HTML type defs)
- raster2-react.js - HTML runtime code (identical to raster2.js)
- raster2.d.ts - HTML type defs
- raster2.js - HTML runtime code

I think you’ll also want to include LICENSE.txt and template.html